sync/class.jetpack-sync-actions.php

```diff
@@ -20,6 +21,9 @@ static function init() {
 
 		if ( self::sync_via_cron_allowed() ) {
 			self::init_sync_cron_jobs();
+		} else if ( wp_next_scheduled( 'jetpack_sync_cron' ) ) {
```

Comment:
If it's possible to avoid running anything that fetches the cron tasks on every page load - ie avoiding calling `wp_next_scheduled()` on every pageload, it'd be *highly* appreciated.
It would be best if it was only triggered by admin page views, rest api calls, or something that triggers a job requeue... ie. a write-possible request..

When running with Cavalcade this ultimately causes a SQL per page load.